tachyon/zk/plonk/permutation/permutation_proving_key_unittest.cc

```diff
@@ -32,8 +32,7 @@ class PermutationProvingKeyTest : public testing::Test {
 }  // namespace
 
 TEST_F(PermutationProvingKeyTest, Copyable) {
-  constexpr size_t kDegree = 5;
-  ProvingKey expected({Evals::Random(kDegree)}, {DensePoly::Random(kDegree)});
+  ProvingKey expected({Evals::Zero()}, {DensePoly::Random(5)});
```

Comment:
```suggestion
  ProvingKey expected({Evals::Random()}, {DensePoly::Random(5)});
```

Reply:
Since `Evals` tries to generate huge sized polynomial, I use `Evals::Zero()` on purpose! I'll leave the comment why I did like this.